### ISSUE TITLE
Disable (or option to disable) Quit on Close

### ISSUE THREAD

**Issue by thecodemill** [Sequel-Ace/Sequel-Ace#2094]
Currently Sequel Ace will quit after the last/only remaining connection window is closed. Can we please have an option to disable this behaviour?

The number of times I have hit ⌘+W followed by ⌘+N, only to find I'm now composing a new email is getting quite old! 🤣

**Comment by Kaspik** [comment]
Search issue before opening one.

**Comment by thecodemill** [comment]
I did. Twice. Beats me which one you’re referring to though.

**Comment by Kaspik** [comment]
https://github.com/Sequel-Ace/Sequel-Ace/issues/2074
https://github.com/Sequel-Ace/Sequel-Ace/issues/2035
https://github.com/Sequel-Ace/Sequel-Ace/issues/2030
https://github.com/Sequel-Ace/Sequel-Ace/issues/2020
https://github.com/Sequel-Ace/Sequel-Ace/issues/2016

**Comment by Kaspik** [comment]
Fair enough! Ideally we would find a time or someone who would find the bug so we can re-enable it again.

**Comment by thecodemill** [comment]
Or perhaps an obvious message if this isn’t being considered for a fix. I suspect a lot of users will continue to report this otherwise.
